index.md

```diff
@@ -54,8 +54,10 @@ for each level and a very brief introduction to plotting.
 
 ## Data
 
-Data files for the lesson are available here: ({{page.dataurl %}})[{{page.dataurl %}}]
+Data files for the lesson are available here: 
+[{{page.dataurl %}}]({{page.dataurl %}})
 
+Please download the **.csv* (Comma Separated Value) files.
```

Comment:
Maybe add that 4 csv files need to be downloaded (species, plots, surveys, and combined).